internal/infrastructure/db/migrations/20230828105616_add_tables.sql

```diff
@@ -37,12 +37,13 @@ CREATE TABLE IF NOT EXISTS pid_configs (
     formula text NOT NULL,
     interval_seconds INTEGER NOT NULL,
     version text NOT NULL,
+    protocol text NOT NULL,
     created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
     updated_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
 
     CONSTRAINT pid_configs_pkey PRIMARY KEY (id)
 );
-CREATE TABLE IF NOT EXISTS power_configs (
+CREATE TABLE IF NOT EXISTS device_settings (
```

Comment:
because this changes an existing migration, before merging and deploying to dev we'll need to rollback the current migration in `dev`